### PR TITLE
Review this dockerfile and come and explain later

### PR DESCRIPTION
**Pull request by graceochie**
Dear all, kindly review this dockerfile and come and explain later to me like a five-year-old

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Use the official Python image from the Docker Hub
+FROM python:3.9-slim
+
+# Set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
```
Comment (ceceteras) [inline Dockerfile:5]: what for?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ENV PYTHONUNBUFFERED 1
```
Comment (ceceteras) [inline Dockerfile:6]: this too

**Comment by Bethel-tech** [comment]
So this looks pretty straight forward, but i'd like more details.

**Comment by graceochie** [comment]
> So this looks pretty straight forward, but i'd like more details.

You're not a serious person-

**Pull request merged**